pkg/controller/windowsmachine/windows/service.go

```diff
@@ -17,30 +12,14 @@ type kubeProxyService struct {
 	args string
 }
 
-// newKubeProxyService returns a service interface with a kubeProxyService implementation
-func newKubeProxyService(nodeName, hostSubnet, sourceVIP string) (service, error) {
-	return &kubeProxyService{
-		binaryPath: kubeProxyPath,
-		name:       kubeProxyServiceName,
-		args: "--windows-service --v=4 --proxy-mode=kernelspace --feature-gates=WinOverlay=true " +
-			"--hostname-override=" + nodeName + " --kubeconfig=c:\\k\\kubeconfig " +
-			"--cluster-cidr=" + hostSubnet + " --log-dir=" + kubeProxyLogDir + " --logtostderr=false " +
-			"--network-name=OVNKubernetesHybridOverlayNetwork --source-vip=" + sourceVIP +
-			" --enable-dsr=false",
+// newService initializes and returns a pointer to the service struct
+func newService(binaryPath, name, args string) (*service, error) {
+	if args == "" || binaryPath == "" || name == "" {
```

Comment:
Args should be able to be empty, as a Windows service being called with no arguments is a valid case